draft-ietf-quic-transport.md

```diff
@@ -2246,6 +2246,14 @@ that packet numbers cannot be used to correlate activity.  This does not prevent
 other properties of packets, such as timing and size, from being used to
 correlate activity.
 
+An endpoint SHOULD NOT initiate migration with a peer that uses a zero-length
+connection ID, for two reasons. First, if the peer routes incoming packets using
+the packets' source address, migration might not be successful. Second, if
+such a  peer routes incoming packets by assigning a unique destination address
+to the connection, which can be achieved using using the preferred_address
+transport parameter (see {{preferred-address}}), packets sent by this
+endpoint over multiple paths are trivially linkable.
```

Comment:
In contrast to my comment on the first part, saying SHOULD NOT do trivially linkable things seems reasonable -- by default, don't, and if you've got some reason to believe that you don't care or can avoid that likability, then that's on you.